amundsen_application/static/js/features/ColumnList/ColumnStats/index.tsx

```diff
@@ -41,8 +42,9 @@ const ColumnStats: React.FC<ColumnStatsProps> = ({
   if (stats.length === 0) {
     return null;
   }
-  const startEpoch = Math.min(...stats.map(getStart));
-  const endEpoch = Math.max(...stats.map(getEnd));
+  const filteredStats = filterOutUniqueValues(stats);
```

Comment:
Just a nit about readability: We're using the both distinct (`distinctStatTypeName`) and unique (`uniqueValues`) somewhat interchangeably. I think it would be better to use either `distinct` or `unique` consistently for both the configs and util functions.

Reply:
I was using the unique values name until I found they call them distinct on the backend.

you are right, I will rename the configuration.